SignalServiceKit/src/Protos/Generated/SSKProto.swift

```diff
@@ -80,6 +80,11 @@ public enum SSKProtoError: Error {
             proto.content = valueParam
         }
 
+        // NOTE: This method is intended for debugging purposes only.
+        @objc public func buildIgnoringErrors() -> SSKProtoEnvelope {
```

Comment:
I've added `buildIgnoringErrors` and `serializedDataIgnoringErrors` because it makes "debug UI" and "test" code dramatically easier.

Reply:
Why not put this in a base SSKProtoWrapper class rather than generate it for every message?

Also if we want them only for debug, it would be good to only conditionally include them.

Reply:
> Why not put this in a base SSKProtoWrapper class rather than generate it for every message?

Want to take this on?  It might be nice to get familiar with how the code generation works.

> Also if we want them only for debug, it would be good to only conditionally include them.

I added a `guard` to that effect.

Reply:
> Want to take this on? It might be nice to get familiar with how the code generation works.

Sure